src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -982,11 +989,22 @@ public void run() {
      * {@inheritDoc}
      */
     @Override
+    protected boolean supportsMakeDisabled() {
+        if (sources == null || sources.isEmpty()) {
+            return false;
```

Comment:
This is actually bad because it means that you cannot mark a folder disabled in its `configure` screen when initially creating it (since the `sources` have not yet been saved), which is a desirable thing to be able to do if you want to get started with the configuration before actually making anything live. So I would suggest removing this override. Same in `OrganizationFolder`.